pulumi/github/repos/README.md

```diff
@@ -0,0 +1,34 @@
+# Repos
+
+Goal is to replace https://oldsite.nf-co.re/pipeline_health
+
+This repo will be the "Actions" section at the bottom. We can then create a reporting page if we really need to see all the green checks
+
+## Initial Roll-out
+
+The new pipelines that are broken:
+
+- denovotranscript
+- meerpipe
+- pairgenomealign
+- phaseimpute
+- reportho
+
+Maybe:
+
+- scdownstream
+- scnanoseq
+
+### Plan
+
+#### Short-term
+
+1. [ ] Import a pipeline that has all the right settings
+2. [ ] Fix the 5 pipelines above with the correct settings from the "model" repo
+3. [ ] Keep importing new pipelines until we gain confidence in it.
+
+#### Long-term
+
+1. Wrangle in `core_repos`
+2. Roll out to all pipelines
+3. Switch all repos to main
```

Comment:
I'd rather track this as its own issue, as it impacts a lot of stuff and needs fairly close coordination. @mashehu and @mirpedrol are already on the case, making tooling work with either `main` or `master` so that we can make the switch.

Reply:
Completely agree, this was more of a catch-all, hence the long-term, not in this PR or project necessarily, more of enabling it.